doc/host-setup.md

```diff
@@ -111,6 +112,28 @@ ENTRYPOINT ["sh", "-e", "/entry-override.sh"]
 Build the image, tag it and push it to a registry accessible to your cluster. You can start using your custom injector
 image by passing `--set operator.satelliteSet.kernelModuleInjectionImage=<your-custom-image>` to Helm.
 
+For Ubuntu 22.04, we supply a `Dockerfile` as an example as well
```

Comment:
Why would you need this file? I mean what is different from https://github.com/piraeusdatastore/piraeus/blob/master/dockerfiles/drbd-driver-loader/Dockerfile.jammy ?

Reply:
I just added the suggested modifications